capstone/client/src/components/Header.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import { AppBar, Toolbar, IconButton, Typography } from '@material-ui/core';
+import { Alert } from '@material-ui/lab';
+import axios from 'axios';
+import { Menu } from '@material-ui/icons';
+import { GoogleLogin, GoogleLogout } from 'react-google-login';
+
+import { Store } from './Store';
+
+const CLIENT_ID = "618901837293-lggv074jcmas0qvt2gvatjsb62r219om.apps.googleusercontent.com";
+
+class Header extends Component {
+    constructor(props) {
+      super(props);
+      this.state = {
+          alert: null,
+          googleButton: this.props.store.get('loggedIn') ? (
+            <GoogleLogout
```

Comment:
I think usually we only store data in state (props and store too), so move the HTML components to the render function? You can store the isLoggedIn bool flag in state and looks like it'll be enough.